lib/root.js

```diff
@@ -15,6 +15,14 @@ process.on('exit', function (code) {
     process.exit(1)
 })
 
+// need to autoend if a teardown is added.
+// otherwise we may never see process.on('exit')!
+tap.tearDown = tap.afterAll = function (fn) {
+  var ret = Test.prototype.afterAll.apply(this, arguments)
+  this.autoend()
```

Comment:
Maybe make `tap.tearDown` root-only method and have `Test.prototype.afterAll` not aliased as `.tearDown` so the same method on instances and pseudo-instances of `Test` are consistent in behaviour?